api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClientImpl.java

```diff
@@ -578,11 +568,10 @@ public List<LeonardoListPersistentDiskResponse> listDisksAsService() {
   }
 
   @Override
-  public List<LeonardoListPersistentDiskResponse> listDisksByProjectAsService(
-      String googleProject) {
+  public List<ListPersistentDiskResponse> listDisksByProjectAsService(String googleProject) {
 
-    var disksApi = legacyServiceDisksApiProvider.get();
-    return legacyLeonardoRetryHandler.run(
```

Comment:
I see that legacyLeonardoRetryHandler is still defined above. Is that necessary?

Reply:
yes.  This PR only updates the DisksApi calls, so the legacy client continues to be needed for calls to AppsApi, RuntimesApi, etc.